deploy/sre-prometheus/ocm-agent/100-ocm-agent.PrometheusRule.yaml

```
@@ -18,4 +18,28 @@ spec:
         severity: Info
         namespace: openshift-logging
         send_managed_notification: "true"
-        managed_notification_template: "LoggingVolumeFillingUp"
\ No newline at end of file
+        managed_notification_template: "LoggingVolumeFillingUp"
+    - alert: ElasticsearchNodeDiskWatermarkReachedSRE
```

Would prefer if all our OAO alerts have `Notification` in them so that it's easier for SRE to identify "this alert is for OAO". (maybe we can add this to our OAO onboarding SOPs?)

```suggestion
    - alert: ElasticsearchNodeDiskWatermarkReachedNotificationSRE
```